.devcontainer/devcontainer.json

```
@@ -0,0 +1,4 @@
+{
+"name": "minimal_ubuntu",
+"image": "ruby:3.0.6"
+}
```

Super minor indent nitpick
```suggestion
{
  "name": "minimal_ubuntu",
  "image": "ruby:3.0.6"
}
```